### PR TITLE
Fix/links

### PR DESCRIPTION
**Pull request by robitalec**
# Prework

* [X] I understand and agree to this repository's [code of conduct](https://ropensci.org/code-of-conduct/).
* [X] I understand and agree to this repository's [contributing guidelines](https://github.com/ropensci-boooks/targets-manual/blob/main/CONTRIBUTING.md).
* [ ] I have already submitted an issue to the [issue tracker](http://github.com/ropensci-boooks/targets-manual/issues) to discuss my idea with the maintainer.

# Related GitHub issues and pull requests

* Ref: #

# Summary

Just a PR fixing some dead links, let me know if you'd like an accompanying issue opened as well. Figured it was small enough to not require an issue as well. 

Thanks as always for the incredible package!

### CODE REVIEW

File: dynamic.qmd [hunk 1]
```diff
@@ -15,24 +15,24 @@ library(targets)
 library(dplyr)
 ```
 
-:::{.callout-tip}
-## Performance
+::: callout-tip
+## Performance {#performance}
```
Comment (wlandau) [inline dynamic.qmd:19]: This might conflict with https://books.ropensci.org/targets/performance.html

File: dynamic.qmd [hunk 2]
```diff
@@ -42,8 +42,9 @@ To use dynamic branching, set the `pattern` argument of `tar_target()`. The patt
...
+To illustrate, consider the example pipeline below. It uses dynamic branching to generate random [spirographs](https://en.wikipedia.org/wiki/Spirograph) using code borrowed from [W. Joel Schneider](https://github.com/wjschne)'s [`spiro`](https://wjschne.github.io/spiro/index.html) package.[^dynamic-1]. A spirograph is a type of two-dimensional algebraic curve determined (in part) by parameters `fixed_radius` and `cycling_radius`. Targets `fixed_radius` and `cycling_radius` draw random parameter values, and the dynamic target `points` generates a spirograph dataset for each set of parameters (one spirograph per dynamic branch). Target `single_plot` plots each spirograph separately, and `combine_plot` plots all the spirographs together.
```
Comment (wlandau) [inline dynamic.qmd:45]: I prefer to keep footnotes inline.

File: dynamic.qmd [hunk 3]
```diff
@@ -167,10 +168,11 @@ Use the `branches` argument of `tar_read()` to read an individual branch or subs
 tar_read(single_plot, branches = 1)
 ```
 
-
 ## Provenance
 
-Recall our dynamic target `points` with branches for spirograph datasets. Each branch has columns `fixed_radius` and `cycling_radius` so we know which parameter set each spirograph used. It is good practice to proactively append this metadata to each branch, e.g. in `spirograph_points()`. That way, if a branch errors out, it is easy to track down the upstream data that caused it.^[See also <https://books.ropensci.org/targets/debugging.html#workspaces>.]
+Recall our dynamic target `points` with branches for spirograph datasets. Each branch has columns `fixed_radius` and `cycling_radius` so we know which parameter set each spirograph used. It is good practice to proactively append this metadata to each branch, e.g. in `spirograph_points()`. That way, if a branch errors out, it is easy to track down the upstream data that caused it.[^dynamic-2]
+
+[^dynamic-2]: See also <https://books.ropensci.org/targets/debugging.html#workspaces>.
```
Comment (wlandau) [inline dynamic.qmd:175]: Likewise for this footnote.

File: walkthrough.qmd [hunk 4]
```diff
@@ -107,7 +107,7 @@ plot_model <- function(model, data) {
...
...and `functions.R` are typical user-defined components of a [project-oriented workflow](https://rstats.wtf/projects), the target script file `_targets.R` file is special. Every [`targets`](https://github.com/ropensci/targets) workflow needs a [target script file](https://docs.ropensci.org/targets/reference/tar_script.html) to configure and define the pipeline.^[By default, the target script is a file called `_targets.R` in the project's [root directory](https://martinctc.github.io/blog/rstudio-projects-and-working-directories-a-beginner%27s-guide/). However, you can set the target script file path to something other than `_targets.R`. You can either set the path persistently for your project using [`tar_config_set()`](https://docs.ropensci.org/targets/reference/tar_config_set.html), or you can set it temporarily for an individual function call using the `script` argument of [`tar_make()`](https://docs.ropensci.org/targets/reference/tar_make.html) and related functions.] The [`use_targets()`](https://docs.ropensci.org/targets/reference/use_targets.html) function in `targets` version >= 0.12.0 creates an initial target script with comments to help you fill it in. Ours looks like this:
```
Comment (wlandau) [inline walkthrough.qmd:110]: I'm not sure what changed here. If you could point it out or remove the diff, that would be great.

File: dynamic.qmd [hunk 5]
```diff
@@ -184,12 +186,12 @@ tar_read(points, branches = 2) # second branch
 
 `targets` supports many more types of dynamic branching patterns.
 
-* `map()`: one branch per tuple of elements.
-* `cross()`: one branch per *combination* of elements.
-* `slice()`: select individual pieces to branch over. For example, `pattern = slice(x, index = c(3, 4))` branches over the third and fourth slices (or branches) of target `x`.
-* `head()`: branch over the first few elements.
-* `tail()`: branch over the last few elements.
-* `sample()`: branch over a random subset of elements.
+-   `map()`: one branch per tuple of elements.
```
Comment (wlandau) [inline dynamic.qmd:189]: I generally prefer the `*` list style.

**Review by wlandau — CHANGES_REQUESTED** [review]
Thanks for fixing the links. The PR looks mostly good, but it would be nice if it were simpler, i.e. just the links. I added specific comments inline.

**Comment by robitalec** [comment]
Thanks @wlandau - apologies for this wasn't meant to be a messy PR.. I flipped into the visual editing to check something and I think RStudio auto changed these. I should've caught them before committing - editing now

File: walkthrough.qmd (hunk 4 above)
Comment (robitalec) [reply walkthrough.qmd:110]: The change is "https://rstats.wtf/projects" from the dead link "https://rstats.wtf/project-oriented-workflow.html"

**Comment by robitalec** [comment]
Thank you, and I appreciate the requested changes!

**Pull request merged**